bookwyrm/settings.py

```diff
@@ -58,6 +58,7 @@
 
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = env.bool("DEBUG", True)
+USE_HTTPS = env.bool("USE_HTTPS", False)
```

Comment:
I'm defaulting `USE_HTTPS` to False, but it might not be best for people who are updating their instance in Production and haven't updated their `.env`. What's your opinion?

Reply:
hm, I think nginx should automatically redirect to https in these cases? I wonder if it will cause a problem for social media previews 🤔

Reply:
Ah yes, I forgot about nginx auto redirection! I don't think it'll be a problem.